integration_tests/utils.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
```

Comment:
Maybe you can create a new file named `podman_lib.sh` and just include it in the test files. Also, keep in mind that some commands may fail; in this sense, add a proper error check. Also, consider using `cmd_manager`, that way, you can also create a verbose option for the integration test.